systemd/cloud-config.service.tmpl

```diff
@@ -2,8 +2,14 @@
 [Unit]
 Description=Apply the settings specified in cloud-config
 After=network-online.target cloud-config.target
+{% if variant not in ["rhel"] %}
 After=snapd.seeded.service
```

Comment:
Any After=X generally won't have applicability/impact on any distro unless they have a snapd.seeded.service installed. Also, since snapd looks like it is installable from the `epel-release` stream, it seems like rhel images could have this installed and I think we still want cloud-config.service to start after the seeding completes.

Reply:
Makes sense, I was not sure what it was and decided to go safe and avoid adding something that could potentially break our downstream.